### PR TITLE
Adds basic warning about the WebKit delay

### LINKED ISSUE #1197: Display message about Safari when in "flutter mode"
Given that we're not likely to get a fix for the Safari/DDC issue in the short term, we need to add a message popup to the new embed that warns people about the issue.

### PR DESCRIPTION
**Pull request by RedBrogdon**
Fixes #1197.

Adds a method to the new embed that sniffs the useragent string for Safari and displays a dialog with a warning if it's detected. This will be removed once we've gotten some kind of resolution on #1108 and https://bugs.webkit.org/show_bug.cgi?id=199866.

This change is staged and can be tested at

https://20190815t141213-dot-dart-pad.appspot.com/experimental/new_embeddings_demo.html

Feel free to bang on it with a variety of browsers.

### CODE REVIEW

File: lib/experimental/new_embed.dart [hunk 1]
```diff
@@ -407,6 +415,27 @@ class NewEmbed {
     await modules.start();
   }
 
+  void _notifyIfWebKit() {
+    // See https://bugs.webkit.org/show_bug.cgi?id=199866.
+    if (window.navigator.vendor.contains('Apple') &&
+        !window.navigator.userAgent.contains('CriOS') &&
+        !window.navigator.userAgent.contains('FxiOS')) {
+      dialog.showOk('Possible delay', '''
```
Comment (johnpryan) [inline lib/experimental/new_embed.dart]: nit: consider just "OK" or "I understand". "Possible delay" sounds like it's enabling a delay.

Comment (RedBrogdon) [reply lib/experimental/new_embed.dart]: Not sure I understand. "Possible delay" is the title text for the dialog.

Comment (johnpryan) [reply lib/experimental/new_embed.dart]: oops. I misread the arguments.

Comment (johnpryan) [reply lib/experimental/new_embed.dart]: LGTM

**Pull request merged**